cost/cloud_cost_anomaly_alerts/README.md

```diff
@@ -21,8 +21,10 @@ Any point outside of the bands is considered as anomalous. If multiple cost anom
 - *Minimum Period Spend* - minimum daily spend needed to record an anomaly as an incident.
   - Minimum Value: 0
   - Default: 1000
-- *Cost Anomaly Dimensions* - Dimensions to group data for the analysis. Multiple dimensions can be provided. Currently, custom tags cannot be used as dimensions.
+- *Cost Anomaly Dimensions* - Dimensions to group data for the analysis. Multiple dimensions can be provided.
   - Default: ["Cloud Vendor Account", "Cloud Vendor", "Service"]
```

Comment:
Dimensions to group data for analysis. It is possible to provide multiple dimensions.

Can we change ?